typeql/reasoner/negation.feature

```diff
@@ -1202,3 +1202,71 @@ Feature: Negation Resolution
       """
     Then verify answer size is: 0
 #    TODO: Add a case with non-zero answers
+
+
+  Scenario: Negated relation which is both retrievable and concludable must always consider both (issue#6500)
+    # In the issue, (from: door, to:common-room) was incorrectly returned, as it contradicts a retrievable.
```

Comment:
You can actually do without the `#` here - Cucumber expects a description at the start of a scenario.